plugins/woocommerce-admin/webpack.config.js

```diff
@@ -168,6 +168,23 @@ const webpackConfig = {
 		},
 	},
 	plugins: [
+		// Workaround for Gutenber private API consent string differences between WP 6.3 and 6.4+
```

Comment:
Typo `Gutenber`

Reply:
Fixed in b972c515fc382572a212b4c546524353c47af58d.